### ISSUE TITLE
freeCodeCamp articles don't get saved to @Pocket correctly

### ISSUE THREAD

**Issue by QuincyLarson** [freeCodeCamp/freeCodeCamp#46501]
**Describe the bug**
When saving freeCodeCamp articles to Pocket, articles don't get saved correctly, and instead it seems to save some more general metadata and say the article's author is "clearTimeout".

**Expected behavior**
Normally, Pocket should save the entire article so people can read it offline.

**Screenshots**
Video with behavior here: https://twitter.com/dk_prabhu/status/1536768123180929027

**Comment by pdwarkanath** [comment]
I reported this.

Phone: Pixel 4a
Browser: Brave

**Comment by ieahleen** [comment]
Pocket guide says ([here](https://help.getpocket.com/article/1077-what-is-article-view)
> Reporting issues with Article View
> If you discover that an item you've saved to Pocket doesn't open in Article View, or has missing or incorrect content in Article View (such as missing images or sentences) please send us the link so we can try to get it fixed.

Someone may try to reach out to Pocket support to see if it's their software that needs adjusting or if something can be done on this side

**Comment by scissorsneedfoodtoo** [comment]
Thanks for reporting this issue and for your help contacting the Pocket team, @pdwarkanath.

I just tried this with the Pocket app on Android and everything seems fine on my end now. I'm able to open an article on Chrome, save it to Pocket, and view the whole article in the Pocket app.

I also tried saving a couple of articles with the Pocket Chrome extension. Again, saving an article works and I'm able to view the whole article from https://getpocket.com/my-list.

Closing this as fixed.
